ansible/roles/invoker/tasks/clean.yml

```diff
@@ -38,3 +38,14 @@
     path: "{{ whisk_logs_dir }}/invoker{{ groups['invokers'].index(inventory_hostname) }}"
     state: absent
   become: "{{ logs.dir.become }}"
+
+# Workaround for orphaned ifstate.veth* files on Ubuntu 14.04
+# See https://github.com/moby/moby/issues/22513
+# Remove inactive files older than 60 minutes
+- name: "Clean orphaned ifstate.veth* files on Ubuntu 14.04"
+  shell: |
+    ACTIVE_IFACES_REGEXP=$(ip -oneline link show | grep --only-matching --extended-regexp 'veth[0-9a-f]+' | tr '\n' '|' | sed -e 's/.$//' | xargs -I '{}' /bin/echo '/run/network/ifstate\.({})')
+    find /run/network -regextype posix-egrep \( -not -regex ${ACTIVE_IFACES_REGEXP} \) -and -name 'ifstate.veth*' -and -mmin +60 -delete
+  ignore_errors: True
```

Comment:
remove the line `ignore_errors` I think we want to catch when there is a problem deleting this files.

Reply:
Okay, will do. I had the feeling that we don't want to hold a deployment due to cleaning problems - but probably you are right. It's better to recognize problems - otherwise they will go unnoticed.

Reply:
Good point. Done.